starknet_py/tests/e2e/fixtures/devnet.py

```diff
@@ -15,19 +17,22 @@ def get_available_port() -> int:
         return sock.getsockname()[1]
 
 
-def start_devnet():
+def start_devnet(get_start_devnet_func, fork_mode: Optional[bool] = False):
     devnet_port = get_available_port()
-    start_devnet_command = get_start_devnet_command(devnet_port)
+    start_devnet_command = get_start_devnet_func(devnet_port, fork_mode=fork_mode)
```

Comment:
I guess there is no need to pass a function as an argument to `start_devnet`
```suggestion
    start_devnet_command = get_start_devnet_command(devnet_port, fork_mode=fork_mode)
```